docs/requirements-docs.txt

```diff
@@ -1,5 +1,6 @@
 sphinx==5.1.1
 sphinx-pypi-upload
 furo==2022.6.21
+git+https://github.com/harshil21/furo-sphinx-search@main
```

Comment:
can we pin a specific commit once we're happy (e.g. right before merge)? Makes me feel safer :D